apps/docs/pages/guides/auth/auth-smtp.mdx

```diff
@@ -0,0 +1,38 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-smtp',
+  title: 'Configure a Custom SMTP',
+  description: 'Moving towards production: Configuring a custom SMTP provider',
+}
+
+## Overview
+
+At present, you can trial the Supabase platform by sending up to four emails per hour via the built service. As you progress toward production, you may find yourself wanting for a custom SMTP service in order to increase your limits. A custom SMTP server will allow you to set your own cap on the number of emails sent per hour.
+Beyond rate limits, an SMTP server might also help with:
+
+- Deliverability and Reputation Management
+- Scalability
+- Analytics and Tracking
+- Compliance and Anti Spam measures
+
+
+## How to Set up SMTP
+
+Head over to the [Auth Settings Page](https://supabase.com/dashboard/project/_/settings/auth) and hit "Enable Custom SMTP" under the SMTP Provider section.
+
+Fill in fields below with the relevant details obtained from your custom SMTP provider:
+
+[SMTP settings](/docs/img/guides/auth-smtp/smtp.png)
```

Comment:
Consider making the username in the screenshot something more like `user@example.com` (we should standardize on this across docs)